### DISCUSSION TITLE
Machinaris support for Chives Coin (XCC)

### DISCUSSION THREAD

**Discussion by WebNetSolving** [guydavis/machinaris#265] [Ideas]
Hi dear "guydevis" can I ask you the courtesy of supporting Chives (XCC) too https://www.chivescoin.org/

We would be very many to be grateful to you.

Regardless of your answer, I thank you in advance for the support and I congratulate you on Machinaris because it is really great.

I hope to receive your reply and thanks again in advance.

:-)

**Comment by guydavis** [comment]
Hi, I'll give it a look. Can't promise anything though as each fork seems to behave slightly differently, particularly as each version progresses over the months.  Thanks for reaching out.

**Reply by WebNetSolving** [reply]
Hi dear "guydevis" and thank you very much for replying very soon.

I thank you in advance for having taken into consideration my proposal even without any commitment of promise on your part because I am sure that with your skills for me you will surely be able to make Machinaris compatible also with Chives (XCC).

I thank you again for your kindness, for your support and for the time you have dedicated to me.

When and if you have it, I look forward to hearing from you here with the hope that soon your excellent Machinaris will be compatible with Chives (XCC) which I am sure would make many users of the Chives Community very happy.

I salute you and wish you a good day and a good job. :-)

**Reply by guydavis** [reply]
Good day!  Just to let you know that [upcoming version of Machinaris v0.6.0](https://www.machinaris.app/), available now in test, includes support for the [Chives blockchain](https://github.com/guydavis/machinaris/wiki/Chives).  Please give it a try!  All feedback is appreciated.

**Reply by guydavis** [reply]
Version 0.6.0 now released. Marking as answered.

**Reply by WebNetSolving** [reply]
Thank you on behalf of the whole Chives Coin XCC Community for your support and your kindness, thank you.

If you want us and the whole Chives Coin XCC Community we would love to have you aboard our Official Server on Discord, I leave you the invitation here:

https://discord.gg/9aAbYNqjtA

Thanks again for your support.
